_pkgdown.yml

```diff
@@ -1,5 +1,8 @@
 url: https://jangorecki.gitlab.io/data.table
 
+development:
+  version_tooltip: "Odd versions are in development; even are on CRAN"
```

Comment:
we can safely use "development version" as we won't publish even releases in CI, even are cran-only